matrix/matrix_operation.py

```diff
@@ -18,6 +18,7 @@ def add(*matrix_s: list[list]) -> list[list]:
         for i in matrix_s[1:]:
             _verify_matrix_sizes(matrix_s[0], i)
         return [[sum(t) for t in zip(*m)] for m in zip(*matrix_s)]
+    raise TypeError("Expected a matrix, got int/list instead")
```

Comment:
Could we please have a doctest that proves this error will be raised?  It should look something like this:
```python
>>> add(<something_problematic_here>)
Traceback (most recent call last):
  ...
TypeError: Expected a matrix, got int/list instead
```

Reply:
Alright, on it